indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -50,6 +43,40 @@
                                    data-title="{% trans %}Delete template{% endtrans %}"
                                    data-update="#email-template-manager">
                                 </a>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="i-box-content">
+                    {% trans %}This email will be sent when the abstract is:{% endtrans %}
+                        <ul>
+                            {% for rule in tpl.rules %}
+                                {% if rule %}
```

Comment:
Can this ever be falsy? Anyway, you can include this in the loop:

    {% for rule in tpl.rules if rule %}